nrpy/infrastructures/BHaH/numerical_grids_and_timestep.py

```diff
@@ -146,7 +146,9 @@ def register_CFunction_numerical_grid_params_Nxx_dxx_xx(
     )
 
 
-def register_CFunction_cfl_limited_timestep(CoordSystem: str) -> None:
+def register_CFunction_cfl_limited_timestep(
+    CoordSystem: str, use_unit_CFL: bool = False
```

Comment:
I'm not a fan of this change, as it's trivial to set CFL_FACTOR to 1 without modifying the C codegen itself. If you want it to always be set to one, the recommended procedure would be to set its default value to one. Just add this line after the imports in your example/*.py file:

```
par.adjust_CodeParam_default("CFL_FACTOR", 1.0)
```
Let me know if you also want it removed from your parameter file; this is not recommended but can be accomplished by deleting the parameter and re-registering with `add_to_parfile=False`.

Reply:
The CFL_FACTOR is set to 1 only when computing dt, the global time step of the relaxation solver. However, CFL_FACTOR is still a steerable parameter, as it's used in `variable_wavespeed_gfs_all_points.c`. The brief explanation is that the Courant factor is absorbed in the definition of the variable wavespeed, as indicated in Eqs. (34) and (B1) of https://arxiv.org/abs/2111.02424.

If you wish to keep the function `register_CFunction_cfl_limited_timestep()` unchanged, I could define another parameter, say, `MINIMUM_GLOBAL_WAVESPEED`, which would be the same parameter $\mathcal{C}_0$ in Eq. (34) mentioned above.

Then the C code would read
```C
in_gfs[IDX4(VARIABLE_WAVESPEEDGF, i0, i1, i2)] = MINIMUM_GLOBAL_WAVESPEED * MIN(dsmin0, MIN(dsmin1, dsmin2)) / dt;
```

The name of the new variable is self-explanatory. However, it would no longer indicate that this constant comes from the absorption of the Courant factor in the variable wave speed definition.

Reply:
I like the idea of creating a new parameter, as CFL_FACTOR is used differently in other NRPy examples.

Reply:
Implemented in https://github.com/nrpy/nrpy/pull/18/commits/eaea0ac8a6a777b1e14873e9d3584d9930ce59e9 and https://github.com/nrpy/nrpy/pull/18/commits/de592380f580a36ea2ea993c12e073924c2230e3. Also added the new parameter at the top of the codegen module in https://github.com/nrpy/nrpy/pull/18/commits/507109ece940e00b220e1f362dbb0c6ed7637291, which should make it easy to figure out where to change it.